src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -29,6 +29,8 @@
   private static final BigDecimal LOWER_SCORE = BigDecimal.valueOf(LOWER_SCORE_VAL);
   private static final Comparator<UserComment> ascendingScoreComparator =
```

Comment:
Since this is no longer used, this comparator can be safely removed.

Reply:
Rick, this comparator is used in categorizeToBucketList to sort userComment

Reply:
Ah, you're right.